OTRExporter/DisplayListExporter.cpp

```diff
@@ -454,9 +454,23 @@ void OTRExporter_DisplayList::Save(ZResource* res, const fs::path& outPath, Bina
 				}
 				else
 				{
-					word0 = 0;
-					word1 = 0;
-					spdlog::error(StringHelper::Sprintf("dListDecl == nullptr! Addr = {:08X}", GETSEGOFFSET(data)));
+					uint32_t seg = data & 0xFFFFFFFF;
```

Comment:
These changes look good but could you add a comment explaining its purpose? Something like `// If we can't find the display list in this file, try looking in other files based on the segment number`